README.md

```diff
@@ -88,9 +88,10 @@ var options = {
 };
 ```
 
-#### `uncaughtException` (*Function*, optional)
+#### `uncaughtException` (*Function*, optional) **DEPRECATED**
 Handler for `uncaughtException` errors. See the [endgame](https://github.com/totherik/endgame) module for defaults.
 
+**Deprecated**: See `shutdown` middleware instead.
```

Comment:
I added some more detail to the `shutdown` middleware part of the documentation to describe the default behavior.